scss/components/_state-screen.scss

```diff
@@ -135,6 +135,36 @@
     left:-3%;
   }
 
+  .oo-info-panel-cast {
+    @extend .oo-center-vertical-horizontal;
+    @include vendor-prefixes (transition, $fade-up-transition);
+    text-align: center;
+    max-width: 80%;
+    border: 1px solid gray;
+    border-radius: 3px;
+    width: 100%;
```

Comment:
Could you tell me, please, why you set width to 100%, but max-width to 80%? Is it a trick for anything?

Reply:
yeah, it's kindly a trick to tell the div to use all the space but not beyond the 80% of the parent width...
If you have another trick/way to get a similar result I'll happy to check it out :)